perseo-main/src/main/java/com/telefonica/iot/perseo/Utils.java

```diff
@@ -20,12 +20,8 @@
 */
```

Comment:
CHANGEX_NEXT_RELEASE entry needed?
New unit tests needed? (maybe properly covered by existing ones, just checking...)

Reply:
In addition, if we have introduced a new configuration parameter, I understand it should be included in the documentation (not remembering right now where Perseo Core documentation is being documented, but I guess some .md file exists in the repository for that).

Reply:
I did consider it. This change does not add any new function. It's just to not cache values from functions.

Reply:
Fixed in [216f68b](https://github.com/telefonicaid/perseo-core/pull/147/commits/216f68b9a75eb51f554d77ca1825d91ab17879a8)